pkgs/top-level/perl-packages.nix

```diff
@@ -1948,6 +1948,20 @@ let
     };
   };
 
+  CDB_File = buildPerlPackage {
+    pname = "CDB_File";
+    version = "0.99";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/T/TO/TODDR/CDB_File-0.99.tar.gz";
+      sha256 = "1c32b30131871e8a7b23f47d8f65d9cdeb87069fa4c221781a03416496f91f16";
+    };
+    meta = {
+      homepage = http://wiki.github.com/toddr/CDB_File;
```

Comment:
- URL returns "HTTP/1.0 400 Bad Request"
- RFC0045 has been approved recently, depreciating unquote URLs: https://github.com/NixOS/rfcs/pull/45

Reply:
I did not notice that, nice catch!